js/src/components/paid-ads/asset-group/asset-field.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { useReducedMotion } from '@wordpress/compose';
+import {
+	useState,
+	useRef,
+	useImperativeHandle,
+	forwardRef,
+} from '@wordpress/element';
+import { chevronUp, chevronDown } from '@wordpress/icons';
+import { Pill } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import AppButton from '.~/components/app-button';
+import HelpPopover from '.~/components/help-popover';
+import './asset-field.scss';
+
+/**
+ * @typedef {Object} AssetFieldHandler
+ * @property {() => void} scrollIntoComponent Scroll to the nearest alignable position to show this component in the visible area.
+ */
+
+/**
+ * Renders an expandable wrapper for editing asset fields.
+ *
+ * @param {Object} props React props.
+ * @param {JSX.Element} props.heading Heading.
+ * @param {JSX.Element} props.subheading Subheading.
+ * @param {JSX.Element} props.help Help content to be shown after clicking on the help icon.
+ * @param {number} props.numOfIssues The number of issues used to label on UI. It only labels when the number is greater than 0.
+ * @param {boolean} [props.initialExpanded=false] Whether the UI to initial with expanded content.
+ * @param {boolean} [props.disabled=false] Whether display the UI in disabled style. It will collapse the content when disabled.
+ * @param {JSX.Element} [props.children] Content to be rendered.
+ * @param {import('react').MutableRefObject<AssetFieldHandler>} ref React ref to be attached to the handler of this component.
+ */
+function AssetField(
+	{
+		heading,
+		subheading,
+		help,
+		numOfIssues,
+		initialExpanded = false,
+		disabled = false,
+		children,
+	},
+	ref
+) {
+	const containerRef = useRef();
+	const [ isExpanded, setExpanded ] = useState( initialExpanded );
+
+	const isReducedMotion = useReducedMotion();
+
+	useImperativeHandle( ref, () => ( {
+		scrollIntoComponent() {
+			containerRef.current.scrollIntoView( {
+				behavior: isReducedMotion ? 'auto' : 'smooth',
+				inline: 'nearest',
+				block: 'nearest',
+			} );
+		},
+	} ) );
+
+	const handleToggle = () => {
+		setExpanded( ! isExpanded );
+	};
+
+	const issuePillText = sprintf(
+		// translators: %d: number of issues in a asset field.
```

Comment:
Typo `an asset field.`

Reply:
Thanks! Fixed in https://github.com/woocommerce/google-listings-and-ads/pull/1853/commits/4c723550d6d9be64d8706cf6ca255a68c0858920.